modules/ROOT/pages/bolt/message.adoc

```diff
@@ -575,6 +581,45 @@ _Servers may choose to include metadata describing the nature of the failure but
 FAILURE {"code": "Example.Failure.Code", "message": "example failure"}
 ----
 
+[[messages-logon]]
+=== Request message `LOGON`
+
+A `LOGON` message carries an authentication request.
+
+*Signature:* `6A`
+
+** The scheme is the authentication scheme.
+Predefined schemes are `none`, `basic`, and `kerberos`.
+** Further entries in the message are passed to the implementation of the chosen authentication scheme.
+Their names, types, and defaults depend on that choice.
+*** The scheme `basic` requires a username `principal::String` and a password `credentials::String`.
+*** The scheme `bearer` merely requires a token `credentials::String` (added in version *4.4*).
+
+If authentication fails, the server responds with a `FAILURE` message and immediately closes the connection.
+
+==== Synopsis
+
+----
+LOGON {scheme::String, ...)
+----
+
+.Example 1
+----
+LOGON {"scheme": "basic", "principal": "user", "credentials": "password"}
+----
+
+.Example 2
+----
+LOGON {"scheme": "basic", "principal": "user", "credentials": "password"}
+----
```

Comment:
Great to have examples! How do the two examples here differ though? :thinking:

Reply:
oops miss copied them -_-